docs/how-to-enable-new-languages.md

```diff
@@ -281,3 +281,130 @@ Once these are in place, you should be able to run `npm run develop` to view you
 
 > [!ATTENTION]
 > While you may perform translations locally for the purpose of testing, we remind everyone that translations should _not_ be submitted through GitHub and should only be done through Crowdin. Be sure to reset your local codebase after you are done testing.
+
+# Deploying New Languages on `/news`
+
+To deploy News for a new language, you'll need to create two PRs. One PR will be to the [CDN repo](https://github.com/freeCodeCamp/cdn), and the other will be to the [News repo](https://github.com/freeCodeCamp/news).
+
+## Prep the CDN Repo for the New Language
+
+News sources trending links and article titles from our CDN during the build and adds them to the footer. News also fetches dayjs files from the CDN during the build to localize dates and times for each language.
+
+### Add a YAML File for Trending Articles
+
+Clone the [CDN repo](https://github.com/freeCodeCamp/cdn) and create a new branch.
+
+In the [`build/universal/trending`](https://github.com/freeCodeCamp/cdn/tree/main/build/universal/trending) directory, create a new file and name it `language.yaml`. For example, if you are launching Dothraki News, name the file `dothraki.yaml`.
+
+Then copy the contents of the [`english.yaml`](https://github.com/freeCodeCamp/cdn/blob/main/build/universal/trending/english.yaml) trending file and paste it into the new YAML file you just created.
+
+The contents will look something like this:
+
+```yaml
+article0title: 'Learn JavaScript'
+article0link: 'https://www.freecodecamp.org/news/learn-javascript-free-js-courses-for-beginners/'
+article1title: 'Linux ln Example'
+article1link: 'https://www.freecodecamp.org/news/linux-ln-how-to-create-a-symbolic-link-in-linux-example-bash-command'
+article2title: 'JS document.ready()'
+article2link: 'https://www.freecodecamp.org/news/javascript-document-ready-jquery-example/'
+article3title: ...
+article3link: ...
+  ...
+```
+
+### Add a dayjs file for the New Language
+
+In the [`build/news-assets/dayjs/1.10.4/locale`](https://github.com/freeCodeCamp/cdn/tree/main/build/news-assets/dayjs/1.10.4/locale) directory, create a new file and name it `isocode.min.js`. For example, if you are launching Dothraki News, name the file `mis.min.js`.
```

Comment:
I don't see this path `build/news-assets/dayjs/1.10.4/locale`. What I see is `build/news-assets/dayjs/days-1.9.1/locale`. Which path is the correct one? @scissorsneedfoodtoo

Reply:
<img width="1384" alt="image" src="https://user-images.githubusercontent.com/1884376/195920738-8902fda8-ab2d-4637-b48c-84dcb9bf4638.png">

Reply:
In any case, we should clear up version no. will change??

Reply:
```suggestion
In the [`build/news-assets/dayjs/<version>/locale`](https://github.com/freeCodeCamp/cdn/tree/main/build/news-assets/dayjs/1.10.4/locale) directory, create a new file and name it `isocode.min.js`. For example, if you are launching Dothraki News, name the file `mis.min.js`.

> [!NOTE]
> The version number will change as the dependencies are updated.
```